packages/stryker/src/reporters/ClearTextReporter.ts

```diff
@@ -7,14 +7,33 @@ const log = log4js.getLogger('ClearTextReporter');
 
 export default class ClearTextReporter implements Reporter {
 
-  constructor(private options: StrykerOptions) { }
+  constructor(private options: StrykerOptions) {
+    this.reporterOptions = options.reporterOptions || {};
+  }
+
+  private reporterOptions: any;
 
   private out: NodeJS.WritableStream = process.stdout;
 
   private writeLine(output?: string) {
     this.out.write(`${output || ''}\n`);
   }
 
+  private outputResult(result: MutantResult) {
```

Comment:
I would rather that the logic of `onAllMutantsTested` gets reused by your code, instead of creating a new similar looking method.